### PR TITLE
Coming up

### LINKED ISSUE #98: Coming up section (MOBILE)
- [ ] Implement the mobile for this section (note the different designs for no events, 1 event, and 2+ events)
- [ ] For 2+ events, the events should be scrollable horizontally 

Screenshot 2022-12-15 at 12 16 46 AM

### PR DESCRIPTION
**Pull request by jhylisa666**
Fixes #98 .

Changelist:

- Made coming up section responsive
- implemented design for multiple events, 1 event, and no events

Screenshots & Screencasts:

https://user-images.githubusercontent.com/92692037/208705467-a154ec2a-e7ac-4626-a83b-3737e60f6c93.mov

### CODE REVIEW

File: live-site/components/coming-up/ComingUp.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    @media ${max.tablet} {
+      padding-left: 1em;
+      padding-bottom: 1em;
+    }
+  }
+`;
+
+const StyledTime = styled.div`
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.styles.tsx]: Can make this one liner margin: 0.5em 0;

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Thanks for working on this kween this is looking FABULOUS!!!!<3 

Just a few comments!
- When you scroll to far to the right this happens
Screenshot 2022-12-20 at 3 25 05 PM
- THe farthest you can scroll without causing the ^ above issue is this far but we can't see the last event - so make sure the user can scroll all the way and see the entire last event
Screenshot 2022-12-20 at 3 25 45 PM

File: live-site/components/coming-up/ComingUp.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <StyledTextContainer>
+              <StyledHeader>{event.header}</StyledHeader>
+              <StyledTime>{event.time}</StyledTime>
+              <StyledBody>{event.body}</StyledBody>
+            </StyledTextContainer>
+          </StyledEvent>
+        </StyledOneEventContainer>
+      </StyledSectionContainer>
+    );
+  } else {
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.tsx]: you can get rid of the else statement and just return it since this is the last case

**Review by Dhivas-Sugumar — APPROVED** [review]
LGTM 🌟 just Judy's changes

**Comment by jhylisa666** [comment]
> Thanks for working on this kween this is looking FABULOUS!!!!<3
> 
> Just a few comments!
> 
> * When you scroll to far to the right this happens
> 
> Screenshot 2022-12-20 at 3 25 05 PM
> 
> - THe farthest you can scroll without causing the ^ above issue is this far but we can't see the last event - so make sure the user can scroll all the way and see the entire last event
> Screenshot 2022-12-20 at 3 25 45 PM

oke. I think I checked and that is smaller than the smallest mobile size in the drop down, but I made it smaller nevertheless!

File: live-site/components/coming-up/ComingUp.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  }
+  @media ${max.tabletSm} {
+    font-size: 0.75em;
+  }
+`;
+
+const StyledBody = styled.div`
+  font-size: 1em;
+  line-height: 1.3em;
+  color: WHITE;
```
Comment (jzhzhang66) [inline live-site/components/coming-up/ComingUp.styles.tsx]: use colors.WHITE

**Review by jzhzhang66 — COMMENTED** [review]
Approving with minor comments! Please fix those before you merge <333 THANKS LISA

**Review by jzhzhang66 — APPROVED** [review]
LGTM

**Pull request merged**